src/Apps/W1/EDocument/App/src/Processing/Import/Purchase/PurchaseOrderMatching/EDocPOMatchingSetup.Table.al

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.eServices.EDocument.Processing.Import.Purchase;
+
+using Microsoft.Purchases.Vendor;
+
+table 6116 "E-Doc. PO Matching Setup"
```

Comment:
I would think this should be living on the vendor page somehow. 
So far we have added the configuration there. I guess a separate table is okay, the field on the vendor page could always be a flow field?

Not sure where the UI for this is.